src/EDI/Parser.php

```diff
@@ -326,7 +329,7 @@ public function load(string $location): self
 	 * @param string $txt
 	 * @return self
 	 */
-    public function loadString(string &$txt): self
+    public function loadString(string $txt): self
```

Comment:
Maybe this reduced the required memory? 🤷

Reply:
I think because I already had the string loaded in a variable? I don't remember :D

Reply:
Well, if it was for performance reasons, it's probably not necessary: http://schlueters.de/blog/archives/125-Do-not-use-PHP-references.html